fileIO.py

```diff
@@ -15,6 +18,18 @@ def __init__(self):
         self.changesA = changeSet.ChangeSet()
     
     def diffFiles(self, iFileA, iFileB ):
+
+        fileA_base_name = ntpath.basename(iFileA)
+        fileB_base_name = ntpath.basename(iFileB)
+
+        if not utilities.valid_file_ext(iFileA):
+            print("\n[-]", fileA_base_name, ": Unacceptable file type\n")
+            return pmEnums.RESULT.BADFILE
+
+        if not utilities.valid_file_ext(iFileB):
+            print("\n[-]", fileB_base_name, ": Unacceptable file type\n")
+            return pmEnums.RESULT.BADFILE
```

Comment:
good implementation and enumeration, we will have this send a pop up box after that gets implemented too.